content/en/security/application_security/troubleshooting.md

```diff
@@ -564,28 +564,41 @@ See the Application Security [set up documentation][11] to validate you you are
 
 Ensure the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` environment variable (`DD_TRACE_TELEMETRY_ENABLED` for NodeJS) is set to `true`, or the corresponding system property for your language is enabled. For example in Java: `-Ddd.instrumentation.telemetry.enabled=true`
 
-## Disabling Threat Detection and Protection
```

Comment:
Can we call it threat management (to make sure we are aligned with the pricing page)?